pkg/config/config_template.yaml

```diff
@@ -4109,6 +4109,15 @@ api_key:
     #
     # enabled: true
 
+    ## @param compute_top_level_by_span_kind - boolean - optional - default: true
+    ## @env DD_OTLP_CONFIG_TRACES_COMPUTE_TOP_LEVEL_BY_SPAN_KIND - boolean - optional - default: true
```

Comment:
Not sure if I'm missing it, but I don't see where this env var is being bound to this config?

Reply:
I believe this happens in this `BindEnv` call: https://github.com/DataDog/datadog-agent/pull/22163/files#diff-f416ebca55ddc51657af5f29c16b4bb7bdb14e14048581e4e54dc6328803eab9R73

Reply:
Since you're using `BindEnv` I don't think you're setting a default. Also, should a beta be enabled by default ?

Reply:
Just updated it to set default to true. The default env variable name gets autogenerated here: https://github.com/DataDog/datadog-agent/blob/f6d189874be0ca95707f3b3c1d763afd679a8e13/pkg/config/model/viper.go#L447-L451